doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -46,6 +46,8 @@
   [#1176](https://github.com/eclipse-iceoryx/iceoryx2/issues/1176)
 * Add option to provide custom `iceoryx2-pal-posix`
   [#1176](https://github.com/eclipse-iceoryx/iceoryx2/issues/1176)
+* Enable Bazel `bzlmod` support for iceoryx2 builds
+  [#1179](https://github.com/eclipse-iceoryx/iceoryx2/pull/1179)
```

Comment:
```suggestion
  [#1179](https://github.com/eclipse-iceoryx/iceoryx2/issues/1179)
```

Reply:
@dkroenke there is no issue for this PR. The only one that somewhat fits is #3 but that one is a stretch.

Reply:
@elBoberido Yes right, we could use https://github.com/eclipse-iceoryx/iceoryx2/issues/355 instead?

```suggestion
  [#355](https://github.com/eclipse-iceoryx/iceoryx2/issues/355)
```